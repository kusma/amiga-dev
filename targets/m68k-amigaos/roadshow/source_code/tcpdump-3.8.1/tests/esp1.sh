#!/bin/sh

uudecode 02-sunrise-sunset-esp.puu

echo -n test esp1...
if (../tcpdump -t -n -E "0x12345678@192.1.2.45 3des-cbc-hmac96:0x4043434545464649494a4a4c4c4f4f515152525454575758" -r 02-sunrise-sunset-esp.pcap | diff - esp1.out)
then
	echo passed.
else
	echo failed.
fi

