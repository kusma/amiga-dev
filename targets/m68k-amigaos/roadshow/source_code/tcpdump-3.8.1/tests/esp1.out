192.1.2.23 > 192.1.2.45: ESP(spi=0x12345678,seq=0x1): 192.0.2.1 > 192.0.1.1: icmp 64: echo request seq 1280 (DF) (ipip-proto-4)
192.1.2.23 > 192.1.2.45: ESP(spi=0x12345678,seq=0x2): 192.0.2.1 > 192.0.1.1: icmp 64: echo request seq 1536 (DF) (ipip-proto-4)
192.1.2.23 > 192.1.2.45: ESP(spi=0x12345678,seq=0x3): 192.0.2.1 > 192.0.1.1: icmp 64: echo request seq 1792 (DF) (ipip-proto-4)
192.1.2.23 > 192.1.2.45: ESP(spi=0x12345678,seq=0x4): 192.0.2.1 > 192.0.1.1: icmp 64: echo request seq 2048 (DF) (ipip-proto-4)
192.1.2.23 > 192.1.2.45: ESP(spi=0x12345678,seq=0x5): 192.0.2.1 > 192.0.1.1: icmp 64: echo request seq 2304 (DF) (ipip-proto-4)
192.1.2.23 > 192.1.2.45: ESP(spi=0x12345678,seq=0x6): 192.0.2.1 > 192.0.1.1: icmp 64: echo request seq 2560 (DF) (ipip-proto-4)
192.1.2.23 > 192.1.2.45: ESP(spi=0x12345678,seq=0x7): 192.0.2.1 > 192.0.1.1: icmp 64: echo request seq 2816 (DF) (ipip-proto-4)
192.1.2.23 > 192.1.2.45: ESP(spi=0x12345678,seq=0x8): 192.0.2.1 > 192.0.1.1: icmp 64: echo request seq 3072 (DF) (ipip-proto-4)
